tools/fleetctl-docker/Dockerfile

```diff
@@ -13,6 +13,7 @@ ARG binpath=build/binary-bundle/linux/fleetctl
 RUN apt-get update \
   && dpkg --add-architecture i386 \
   && apt update \
+  && apt upgrade \
```

Comment:
From the CI failure looks like it needs `-y`?
https://github.com/fleetdm/fleet/actions/runs/5427813048/jobs/9871422710?pr=12610